Source/DafnyCore/Auditor/Assumption.cs

```diff
@@ -103,10 +103,18 @@ bool allowedInLibraries
       allowedInLibraries: false);
   }
 
-  public static AssumptionDescription AssertOnly = new(
+  public static readonly AssumptionDescription AssertOnly = new(
     issue: "Assertion has explicit temporary [{:only}] attribute.",
-    mitigation: "Remove the [{:only}] attribute",
-    mitigationIETF: "Must remove the [{:only}] attribute",
+    mitigation: "Remove the [{:only}] attribute.",
+    mitigationIETF: "Must remove the [{:only}] attribute.",
```

Comment:
```suggestion
    mitigationIETF: "MUST remove the [{:only}] attribute.",
``` 
That's the standard IETF way of writing obligatory requirements.